### PR TITLE
fix(blog): Building multilingual sites with Gatsby

### PR DESCRIPTION
**Pull request by muescha**
## Description

_NOTE: i split to to separate PR - see last comment_

adjustments:
- fix relative path on internal links
- fix codeblock languages
- add code fences at some constants
- changed all intends to 2 tabs
- add code line from previous example, to show where variable `person` comes from

## Related Issues

- #21517 `(blog) Bulding multilingual sites with Gatsby`

### CODE REVIEW

**Comment by muescha** [comment]
i will split this into separate PR for easy check.


adjustments:
- fix relative path on internal links:
  #22924 `fix(blog): multilingual sites: remove local domain`

- fix codeblock languages
- add code fences at some constants
- changed all intends to 2 tabs
- add code line from previous example, to show where variable `person` comes from

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md [hunk 1]
```diff
@@ -115,54 +115,56 @@ If your website is not Google Maps, people most likely won't share their locatio
 
 As I mentioned already, the intranet app contains profile pages for all employees. They are generated dynamically because, well, the list of employees is also dynamic. This code piece that sits in `gatsby-node.js`'s `createPages` generates pages in the original implementation:
 
-```js
+```graphql
```
Comment (laurieontech) [inline docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md:118]: Don't think we should split this one up. It should likely be kept together and use jsx as the language.

**Comment by laurieontech** [comment]
Also, since this was left open and the others are duplicates of smaller changes I'm going to close those. 
In the future, if you break things up please close the umbrella issue.

**Comment by muescha** [comment]
in https://github.com/gatsbyjs/gatsby/pull/22205#issuecomment-598469219 you said i should split it into smaller packages to make PR easy to check 

> open tightly-scoped PRs for each chunk of work, such as a section of the tutorial, to make it easier for us to review and test

and i agree it is more easy to check the small chunks

but why you closed the smaller ones? *i am confused*

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md (hunk 1 above)
Comment (muescha) [reply docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]: changed

**Comment by laurieontech** [comment]
You're correct. However, this PR was first and stayed open. Therefore I reviewed this prior to looking at the smaller ones. So I closed the ones that were no longer relevant.

Going forward you should close the larger PR when you make the smaller ones so that there aren't duplicates.

**Comment by muescha** [comment]
ok :)

👍

**Comment by laurieontech** [comment]
Looks good!

**Pull request merged**